### PR TITLE
add make targets

### PR DESCRIPTION
**Pull request by aljo242**
## Description

Adds new `make` targets:
- `make format`
- `make lint`

These targets help with maintainability end for enforcing better style over time.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://go.dev/blog/godoc).
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Comment by aljo242** [comment]
Only error is CodeCov

File: Makefile [hunk 1]
```diff
@@ -116,6 +116,21 @@ clean:
 
 .PHONY: go-mod-cache clean
 
+FIND_ARGS := -name '*.go' -type f -not -name '*.pb.go'
+
+## format: Run gofmt and goimports.
+format:
+	@echo Formatting...
+	@find . $(FIND_ARGS) | xargs gofmt -d -s
+	@find . $(FIND_ARGS) | xargs goimports -w -local github.com/tendermint/fundraising
```
Comment (jaybxyz) [inline Makefile:125]: What commands do we expect for MacOS devs to use for the target?

**Review by jaybxyz — COMMENTED** [review]
Thanks for adding these two useful targets!

File: Makefile (hunk 1 above)
Comment (aljo242) [reply Makefile:125]: I think the shell command itself will be fine.  I'm using `find` and `xargs` in a way that is compatible across both OS.

Comment (jaybxyz) [reply Makefile:125]: Oh, it was not working in my MacOS but it is working fine now.

**Review by jaybxyz — APPROVED** [review]

**Comment by jaybxyz** [comment]
@aljo242 Before we merge this PR, can you change all targets that use `starport` to `ignite` in `Makefile`?

**Pull request merged**